VimeoUpload/Descriptor System/Descriptor.swift

```diff
@@ -35,7 +35,7 @@ public enum DescriptorState: String
     case Finished = "Finished"
 }
 
-public class Descriptor: NSObject, NSCoding
+open class Descriptor: NSObject, NSCoding
```

Comment:
should all of these open methods be changed back to public?  Do we ever subclass this Descriptor class?